src/openms/include/OpenMS/COMPARISON/SPECTRA/SpectrumAlignment.h

```diff
@@ -51,9 +51,10 @@ namespace OpenMS
   /**
       @brief Aligns the peaks of two spectra
 
-        Using a banded (width via 'tolerance' parameter) alignment.
+        Using a banded (width via 'tolerance' parameter) alignment if absolute tolerances are given.
```

Comment:
can this be done in Seqan in like 2 lines of code?! (I guess it can, never tried tough).. we can also expect Seqan to be a lot faster